src/sqsd.groovy

```diff
@@ -68,8 +68,8 @@ try {
         def messages = sqs.receiveMessage(receiveMessageRequest).getMessages()
         println "Received Messages : " + messages.size()
 
-        // Break when empty
-        if(messages.size() <= 0) break
+        // Break when empty if not running daemonized
+        if(messages.size() <= 0 && config.sqsd.run_daemonized < 1) break
```

Comment:
Yes, that's what I mean. I am researching as we speak the configuration of cron jobs using this baseimage: http://phusion.github.io/baseimage-docker/ . However, @abdielou suggested we could use (quartz)[http://quartz-scheduler.org/] that is much more reliable and easy to implement. We are open to comments about this, since implementing this daemonized should be a nice option for all its users.